webpack/stories/index.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 
 /* eslint-disable no-unused-vars */
```

Comment:
is this comment needed? I don't see any unused vars (I think linkTo was the unused one?)